### PR TITLE
Add Extreme Extermination Chamber (WIP)

### PR DESCRIPTION
**Pull request by kuba6000**
### WORK IN PROGRESS
Insert powered spawner to GUI slot, and it works 💯 

Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10168
Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7979

## TODO

- [ ] Add armor drops
- [ ] Make NEI GUI better
- [ ] Maybe normal GUI too ?
- [ ] Check in full pack and fix possible errors with modded mobs
- [ ] Finish structure

image
image
image
image
image
image
image

### CODE REVIEW

File: src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_ExtremeExterminationChamber.java [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+        return STRUCTURE_DEFINITION;
+    }
+
+    @Override
+    protected GT_Multiblock_Tooltip_Builder createTooltip() {
+        GT_Multiblock_Tooltip_Builder tt = new GT_Multiblock_Tooltip_Builder();
+        tt.
+            addMachineType("Powered Spawner").
+            addInfo("Controller block for Extreme Extermination Chamber").
+            addInfo("Spawns and Exterminates monster's for you").
```
Comment (querns) [inline src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_ExtremeExterminationChamber.java:86]: Minor typo: `monster's` should be `monsters`, with no apostrophe.

**Comment by Cinobi** [comment]
Its all nice and good, buuuuuut.....
The main reason mob farms exists its infernalmobs and thaumcraftspecial drops, will ur spawner work with this loottables too?

**Comment by kuba6000** [comment]
> Its all nice and good, buuuuuut..... The main reason mob farms exists its infernalmobs and thaumcraftspecial drops, will ur spawner work with this loottables too?

Yes

**Comment by boubou19** [comment]
Also, the drops added to TF mobs shouldn't be obtainable (alu ingots, arsenic,...)

**Comment by Florexiz** [comment]
Probably infernal support should be toggleable with speed/time difference

**Pull request closed without merging**